GVRf/Framework/src/org/gearvrf/GVRXMLParser.java

```diff
@@ -183,11 +183,16 @@
                             } else if (attributeName.equals("resolveDepth")) {
                                 settings.eyeBufferParms.enableResolveDepth(Boolean
                                         .parseBoolean(attributeValue));
-                            } else if (attributeName.equals("resolution")) {
+                            } else if (attributeName.equals("resolutionWidth")) {
                                 int resolution = Integer
                                         .parseInt(attributeValue);
```

Comment:
A little bit picky here, the name should be resolutionWidth and resolutionHeight here